samples/image-classifier-augmented-manifest-car-damage-detector.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: sagemaker.aws.amazon.com/v1
+kind: TrainingJob
+metadata:
+  name: car-damage-detector
```

Comment:
This training job name seems inconsistent with the contents. Could we instead name it with regards to the image classification?

Reply:
+1, it looks to me this will better serve customers if it were named `augmented-s3-manifest-image-classifier` or similar.

Reply:
Happy with the rename,